### PR TITLE
Add uiSchema test for unique keys

### PR DESCRIPTION
**Pull request by BarryPiccinni**
This PR removes duplicate questions from the CYA structure.

It also adds a test which checks for duplicate questions, and fails if they keys in the structure are not unique.

Automated tests pass
Lint passes
Manual test passes

### CODE REVIEW

**Review by armoj — CHANGES_REQUESTED** [review]

File: test/uiSchema.test.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        describe('summaryStructure', () => {
+            const structure = pageProperties.summaryStructure;
+            describe('Question IDs', () => {
+                it('Should appear only once in the structure', () => {
+                    let keyArr = [];
+                    structure.forEach(section => {
+                        keyArr = [...keyArr, ...Object.keys(section.questions)];
+                    });
+
+                    const output = findDuplicates(keyArr);
```
[the remark this answers is not in the document]
Comment (armoj) [reply test/uiSchema.test.js:29]: returns a Boolean

File: test/uiSchema.test.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+const findDuplicates = arr => {
+    const sortedArr = arr.slice().sort();
+    const results = [];
+    sortedArr.forEach((item, index) => {
+        if (sortedArr[index + 1] === item) {
+            results.push(item);
+        }
+    });
+    return results;
+};
```
[the remark this answers is not in the document]
Comment (BarryPiccinni) [reply test/uiSchema.test.js:14]: I considered this, but given we're working with an enormous list of strings, I felt a boolean was insufficient. The benefit of this implementation is it returns an array of the duplicate functions, so you can obtain the duplicate id straight from the fail message.

[the remark this answers is not in the document]
Comment (armoj) [reply test/uiSchema.test.js:14]: Ah, yeah I was just thinking of the binary answer here.
